src/components/common/Tabs/Tabs.tsx

```diff
@@ -45,7 +45,6 @@ const Wrapper = styled.div`
     width: 100%;
   }
 `
```

Comment:
What about adding a empty `<HelperBox />` in line 80? So we can have this empty box/container and fill when we need it with some componetts (e.g, search, paginator, filter, etc) 
```
const HelperBox = styled.div`
  width: 100%;
  padding: 16px;
  gap: 16px;
  display: flex;
  align-items: center;
  justify-content: space-between;
`
```

Reply:
I will continue on this path because I've not been able to get the content of the tab to fill the entire space.

![Selection_295](https://user-images.githubusercontent.com/4270166/128234010-234a5739-81a3-4db4-ba06-7de84a087614.png)